### PR TITLE
⚡️Move conditional_availability_page_search_extension from fond-of-spryker to fond-of-impala

### PR DESCRIPTION
**Pull request by gengjozsef**

### CODE REVIEW

File: bundles/conditional-availability-page-search-extension/LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2021-present FOND OF GmbH
```
Comment (julianzimmermann) [inline bundles/conditional-availability-page-search-extension/LICENSE:3]: 2023

**Review by julianzimmermann — APPROVED** [review]

File: bundles/conditional-availability-page-search-extension/phpstan.neon [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parameters:
+    level: 5
+    reportUnmatchedIgnoredErrors: false
+    scanDirectories:
+        - src/Orm/Zed
+    ignoreErrors:
+        - '#.* on an unknown class Generated\\(Client|Glue|Service|Yves|Zed)\\Ide\\AutoCompletion.*#'
```
Comment (daniel-rose) [inline bundles/conditional-availability-page-search-extension/phpstan.neon:7]: Remove this lines

File: bundles/conditional-availability-page-search-extension/composer.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  "autoload": {
+    "psr-4": {
+      "FondOfImpala\\": "src/FondOfImpala/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "FondOfImpala\\": "tests/FondOfImpala/",
+      "Generated\\":"src/Generated/",
+      "Orm\\":"src/Orm/"
```
Comment (daniel-rose) [inline bundles/conditional-availability-page-search-extension/composer.json:34]:
```suggestion
      "FondOfImpala\\": "tests/FondOfImpala/",
      "Generated\\": "src/Generated/",
      "Orm\\": "src/Orm/"
```

**Review by daniel-rose — APPROVED** [review]

File: bundles/conditional-availability-page-search-extension/LICENSE (hunk 1 above)
Comment (gengjozsef) [reply bundles/conditional-availability-page-search-extension/LICENSE]: @julianzimmermann  done

File: bundles/conditional-availability-page-search-extension/phpstan.neon (hunk 2 above)
Comment (gengjozsef) [reply bundles/conditional-availability-page-search-extension/phpstan.neon]: @daniel-rose i've removed the mentioned line

File: bundles/conditional-availability-page-search-extension/composer.json (hunk 3 above)
Comment (gengjozsef) [reply bundles/conditional-availability-page-search-extension/composer.json]: @daniel-rose i made the required changes

**Pull request merged**